npm/webpack-dev-server/src/errors.ts

```diff
@@ -1,66 +1,33 @@
-export interface MissingDependency {
-  prettyName: string
-  packageName: string
-}
-
-const peerDeps: MissingDependency[] = [
-  {
-    prettyName: 'Webpack Dev Server',
-    packageName: 'webpack-dev-server',
-  },
-  {
-    prettyName: 'Html Webpack Plugin',
-    packageName: 'html-webpack-plugin',
-  },
-  {
-    prettyName: 'Webpack',
-    packageName: 'webpack',
-  },
-]
-
-const makePrettyLibs = (libs) => {
-  return libs.reduce((acc, curr, idx) => {
-    acc.prettyNames.push(curr.prettyName)
-    acc.packageNames.push(curr.packageName)
-
-    if (idx >= libs.length) return acc
-
-    return {
-      prettyNames: acc.prettyNames.join(', '),
-      packageNames: acc.packageNames.join(' '),
-    }
-  }, { prettyNames: [], packageNames: [] })
-}
+import packageJson from '../package.json'
 
 export class MissingPeerDependency extends Error {
```

Comment:
I am not sure to do we need this class later @JessicaSachs ? I see that this is `throw` but why? Do we really need to make a custom instance, BTW thanks to throwing we display this error always twice.